server/mailet/rate-limiter/mailetcontainer.xml

```diff
@@ -101,6 +101,14 @@
         </processor>
 
         <processor state="relay" enableJmx="true">
+            <mailet matcher="All" class="GlobalRateLimit">
+                <keyPrefix>myPrefix</keyPrefix>
+                <duration>1h</duration>
+                <count>100</count>
+                <recipients>200</recipients>
+                <size>1000M</size>
+                <totalSize>2000M</totalSize>
+            </mailet>
```

Comment:
Idem we should put "exceededProcessor" in the example.

We should add a processor for sending a Bounce "oups rate exceeded!"